css/pages/index.css

```diff
@@ -1,9 +1,44 @@
+*{
+  box-sizing: border-box;
+}
+
 .container {
   max-width: 750px;
   text-align: center;
   font-size: 1.5em;
 }
 
+.cards-title
+{
+  margin-bottom: 2rem;
+}
+
+.cards
+{
+  display: flex;
+  justify-content: space-between;
+}
+
+.card{
+  border: 1px solid #f3efef;
+  box-shadow: 0 1.5px 10px 0 rgba(0, 0, 0, 0.14);
+  border-radius: 2px;
+  width: 14rem;
+}
+
+.content
+{
+  padding: 18px 5px 18px 5px;
```

Comment:
Since we're using the same padding (18px) for `top and bottom` and same padding (5x) for `left and right`, we can use shorthand syntax here 🙂 

Source: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

![image](https://user-images.githubusercontent.com/56217868/109760985-afc4cd80-7c15-11eb-960f-b0641afb5220.png)
![image](https://user-images.githubusercontent.com/56217868/109761060-cf5bf600-7c15-11eb-83ca-355f3e566528.png)

Reply:
> Since we're using the same padding (18px) for `top and bottom` and same padding (5x) for `left and right`, we can use shorthand syntax here 🙂
> 
> Source: https://developer.mozilla.org/en-US/docs/Web/CSS/padding
> 
> ![image](https://user-images.githubusercontent.com/56217868/109760985-afc4cd80-7c15-11eb-960f-b0641afb5220.png)
> ![image](https://user-images.githubusercontent.com/56217868/109761060-cf5bf600-7c15-11eb-83ca-355f3e566528.png)

Okay ! :)